admin_ui/backend/api/outbound.py

```
@@ -0,0 +1,771 @@
+
+    wav_bytes = _ulaw_to_wav_bytes(ulaw_data)
+    return Response(content=wav_bytes, media_type="audio/wav")
+
+
+@router.get("/campaigns/{campaign_id}/consent/preview.wav")
+async def preview_consent_wav(campaign_id: str):
+    store = _get_outbound_store()
+    try:
+        campaign = await store.get_campaign(campaign_id)
+    except KeyError:
+        raise HTTPException(status_code=404, detail="Campaign not found")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)